src/utils.ts

```diff
@@ -87,10 +87,21 @@ const isRemoteConfigAvailable = (key: string) =>
 
 export const isABTestingAvailable = (key: string): Promise<boolean> => {
```

Comment:
Perhaps we should move this timeout logic to postMessageToNativeApp function

Reply:
for the moment I'd include it only for remoteConfig